src/components/Resume.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import { createPortal } from 'react-dom';
+import { useNavigate } from 'react-router-dom';
+import { Document, Page } from 'react-pdf/dist/esm/entry.webpack5';
+
+import { cromanResume } from '../../public';
+
+import 'react-pdf/dist/esm/Page/TextLayer.css';
+import 'react-pdf/dist/esm/Page/AnnotationLayer.css';
+
+const ModalContent = ({ onClose }) => {
+  return (
```

Comment:
At this point, you don't need to follow my coding patterns, but I would generally write this as an automatic return since nothing is happening between the function declaration and the `return` statement: `const ModalContent = ({ onClose }) => (<div >...</div>);`